examples/examples_nodejs_test.go

```diff
@@ -14,6 +15,12 @@ func TestAccDatabase(t *testing.T) {
 	test := getJSBaseOptions(t).
 		With(integration.ProgramTestOptions{
 			Dir: path.Join(getCwd(t), "database"),
+			// The default character set for the MySQL Docker image has diverged from the upstream provider's value.
+			// Since the upstream provider has been archived and will not be updated, we'll simply expect changes after
+			// creating a DB with the default options:
+			AllowEmptyPreviewChanges: true,
+			AllowEmptyUpdateChanges:  true,
+			ExpectRefreshChanges:     true,
```

Comment:
this is more of a question than a comment 😂 but I presume it is not possible/advisable to recreate the *test* resources to match the state of the provider at archive?

Reply:
> this is more of a question than a comment 😂 but I presume it is not possible/advisable to recreate the *test* resources to match the state of the provider at archive?

This test result is *nothing* to do with the archived provider. It's due to a change in the upstream container